torch/_dynamo/variables/misc.py

```diff
@@ -635,6 +635,29 @@ def call_method(
         return super().call_method(tx, name, args, kwargs)
 
 
+class MethodWrapperVariable(variables.ConstantVariable):
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        if self.value.__name__ == "__get__":
+            assert len(args) == 1 and len(kwargs) == 0
+            return args[0].var_getattr(tx, self.value.__self__.__name__)
```

Comment:
Can't this invoke arbitrary user code?

Reply:
MethodWrappers in particular are bound method wrappers of a C function, you can't reassign `__getattr__` for those.